RedHat_NLP/Deliverable 1.ipynb

```diff
@@ -0,0 +1,285 @@
+              "ok": true,
+              "headers": [
+                [
+                  "content-type",
+                  "application/javascript"
+                ]
+              ],
+              "status": 200,
+              "status_text": ""
+            }
+          },
+          "base_uri": "https://localhost:8080/",
+          "height": 407
+        },
+        "outputId": "a48fb16f-e372-4715-fa68-25c9241a3437"
+      },
+      "source": [
+        "# Web scraping (Kyle, due 02/26/2021)\n",
+        "#     BeautifulSoup for web scraping\n",
+        "\n",
+        "from urllib.request import urlopen\n",
+        "from bs4 import BeautifulSoup\n",
+        "from google.colab import files\n",
+        "import pandas as pd\n",
+        "import io\n",
+        "import numpy as np\n",
+        "import dateutil\n",
+        "from dateutil import parser\n",
+        "import textblob\n",
+        "from textblob import TextBlob\n",
+        "import nltk\n",
+        "from textblob import Word\n",
+        "import matplotlib.pyplot as plt\n",
+        "from nltk.stem.snowball import SnowballStemmer\n",
+        "nltk.download('punkt')\n",
+        "\n",
+        "# upload file\n",
+        "uploaded = files.upload()\n",
```

Comment:
Its unclear to me what this is doing? Why is the first step to upload a file? What is being uploaded? and from where? I would think the first step would be to read data from the google cloud storage endpoint we provided.

I think you need a step here that reads the list of URL's from `https://gcswebci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/canary-release-openshift-origin-installer-e2e-aws-4.5-cnv`. I'm guessing that list is whats in this csv being uploaded? But that step should be done programatically and part of the notebook.